pkgs/applications/misc/quicksynergy/default.nix

```diff
@@ -0,0 +1,37 @@
+{stdenv, fetchurl, pkgs}:
```

Comment:
Explicitly pass the dependencies here. I.e. replace `pkgs` with `pkgsconfig, gtk2, synergy`. You can basically look at any other package for examples.